app/[lang]/dictionaries/en.json

```
@@ -1,5 +1,5 @@
 {
-    "products": {
-      "cart": "Mohit"
-    }
-  }
\ No newline at end of file
+  "products": {
+    "cart": "Mohit"
+  }
```

why not, you get a flat type then
```suggestion
  "products.cart": "Mohit",
```